### PR TITLE
DCKUBE-183 Make testing more flexible so parameters can be passed to …

### PR DESCRIPTION
**Pull request by wkritzinger-atlassian**
…setup.

### CODE REVIEW

File: src/test/scripts/nfs/etc/sv/nfs/run [hunk 1]
```diff
@@ -1,5 +1,8 @@
 #!/bin/bash
 
+modprobe -v nfs
```
Comment (nanux) [inline src/test/scripts/nfs/etc/sv/nfs/run:3]: What was the difference in NFS behaviour when we didn't load the `kmod` and nfs modules into kernel with `modprobe` before?

Are those modules loaded in the host system, i.e. the node running the pod?

File: src/test/scripts/nfs/nfs-server.yaml.template [hunk 2]
```diff
@@ -51,12 +51,25 @@ spec:
...
-      args: ["-c", "apt -qq update && DEBIAN_FRONTEND=noninteractive apt install -qq -y nfs-kernel-server runit inotify-tools; mkdir -p /etc/sv/nfs ; runsvdir /etc/sv"]
+      args: ["-c", "apt -qq update && DEBIAN_FRONTEND=noninteractive apt install -qq -y nfs-kernel-server runit inotify-tools kmod; mkdir -p /etc/sv/nfs ; runsvdir /etc/sv"]
       volumeMounts:
         - mountPath: /srv/nfs
           name: nfs-export-dir
+        - mountPath: /lib/modules
+          name: lib-modules
+          readOnly: true
+      env:
+        - name: NFS_EXPORT_0
```
Comment (nanux) [inline src/test/scripts/nfs/nfs-server.yaml.template:67]: How is this used?

**Review by nanux — APPROVED** [review]

File: src/test/scripts/nfs/etc/sv/nfs/run (hunk 1 above)
Comment (wkritzinger-atlassian) [reply src/test/scripts/nfs/etc/sv/nfs/run:3]: If another NFS container loaded these modules, or if they are loaded by default on the host system, this is not needed. In Docker for Mac, those modules are not loaded by default on the host VM, so we are loading them here. `modprobe` is a no-op if the module is already loaded, so adding this does not hurt in this case.

File: src/test/scripts/nfs/nfs-server.yaml.template (hunk 2 above)
Comment (wkritzinger-atlassian) [reply src/test/scripts/nfs/nfs-server.yaml.template:67]: Oops, that was a leftover from another container I used, previously. I'll remove it.

**Pull request merged**